### DISCUSSION TITLE
Matchers, Extractors and checks are not found in tempalate.alias

### DISCUSSION THREAD

**Discussion by Bilalqadeer91** [projectdiscovery/nuclei-templates#6487] [Q&A]
While making my own custom template i am facing an issue which gaves output:
[ERR] Error occurred loading template Desktop/nuclei/xss.yaml: yaml: unmarshal errors:
  line 16: field matchers not found in type templates.Alias
  line 22: field extractors not found in type templates.Alias
  line 34: field checks not found in type templates.Alias
[FTL] Could not validate templates: errors occured during template validation

I tried to solve it with different solutions but no success.
I also contact with team on discord

**Comment by ehsandeep** [comment]
@Bilalqadeer91 looks like syntax issue with your template, you need to share the template here or in discord to review it.
